packages/modules/vehicles/ovms/api.py

```diff
@@ -169,21 +169,22 @@ async def _fetch_soc(self,
         if float(self.range) > 1000.0:
             self.range = str(float(self.range) / 10)
 
+        self.kms = float(statusDict['odometer']) / 10
+        self.vehicle12v = statusDict['vehicle12v']
         self.soc_ts = statusDict['m_msgtime_s']
-        log.debug("soc=" + str(self.soc) + ", range=" + str(self.range) + ", soc_ts=" + str(self.soc_ts))
 
-        return int(float(self.soc)), float(self.range), self.soc_ts
+        return int(float(self.soc)), float(self.range), self.soc_ts, float(self.kms), float(self.vehicle12v)
 
 
 # sync function
-def fetch_soc(conf: OVMS, vehicle: int) -> Union[int, float, str]:
+def fetch_soc(conf: OVMS, vehicle: int) -> Union[int, float, str, float, float]:
 
     # prepare and call async method
     loop = asyncio.new_event_loop()
     asyncio.set_event_loop(loop)
 
     # get soc, range from server
     a = api()
-    soc, range, soc_ts = loop.run_until_complete(a._fetch_soc(conf, vehicle))
+    soc, range, soc_ts, kms, vehicle12v = loop.run_until_complete(a._fetch_soc(conf, vehicle))
```

Comment:
```suggestion
    soc, range, soc_ts = loop.run_until_complete(a._fetch_soc(conf, vehicle))
```
Darauf wird nicht mehr zugegriffen.